lightly/api/swagger_rest_client.py

```diff
@@ -61,16 +94,26 @@ def request(
             _request_timeout = self.timeout
 
         # Call RESTClientObject.request
-        return super().request(
-            method=method,
-            url=url,
-            query_params=query_params,
-            headers=headers,
-            body=body,
-            post_params=post_params,
-            _preload_content=_preload_content,
-            _request_timeout=_request_timeout,
-        )
+        try:
+            return super().request(
+                method=method,
+                url=url,
+                query_params=query_params,
+                headers=headers,
+                body=body,
+                post_params=post_params,
+                _preload_content=_preload_content,
+                _request_timeout=_request_timeout,
+            )
+        except ApiException as e:
+            if e.reason in [
+                ApiErrorCode.BAD_REQUEST,
+                ApiErrorCode.CONFLICT,
+                ApiErrorCode.MALFORMED_RESPONSE,
```

Comment:
The reason to originally exclude these from the "prettyprint" was because we were going to eliminate the stack trace. Now that we are keeping the stracktrace but formatting it nicely, we can format it nicely for all errors codes.

Reply:
I removed them.

Reply:
gracias